app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    with open(command[1], "r") as file_in, open(command[2], "w") as file_out:
```

Comment:
The code does not check if the source and destination filenames are the same. According to the task description, the function should do nothing if the user tries to copy a file to itself. Consider adding a condition to handle this case.